starknet-ff/Cargo.toml

```diff
@@ -10,26 +10,29 @@ homepage = "https://starknet.rs/"
 description = """
 Starknet field element type
 """
-keywords = ["ethereum", "starknet", "web3"]
+keywords = ["ethereum", "starknet", "web3", "no_std"]
 
 [package.metadata.docs.rs]
 all-features = true
 
 [dependencies]
-ark-ff = "0.3.0"
-bigdecimal = { version = "0.3.0", optional = true }
-crypto-bigint = "0.4.9"
-hex = "0.4.3"
-num-bigint = { version = "0.4.3", optional = true }
-serde = "1.0.152"
-thiserror = "1.0.30"
+ark-ff = { version = "0.3.0", default-features = false }
+crypto-bigint = { version = "0.4.9", default-features = false }
+hex = { version = "0.4.3", default-features = false }
+serde = { version = "1.0.152", optional = true, default-features = false }
+bigdecimal-no-std = { version = "0.4.0", optional = true, default-features = false }
+num-bigint = { version = "0.4.3", optional = true, default-features = false }
+
+
+[features]
+default = ["std"]
```

Comment:
I think it makes more sense for `serde` to also be enabled by default.

Reply:
ok